snap/patches/default_data_dir.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/util.cpp b/src/util.cpp
+index 57b0419..dd21329 100644
+--- a/src/util.cpp
++++ b/src/util.cpp
+@@ -467,7 +467,7 @@ boost::filesystem::path GetDefaultDataDir()
+     return GetSpecialFolderPath(CSIDL_APPDATA) / "faircoin2";
+ #else
+     fs::path pathRet;
+-    char* pszHome = getenv("HOME");
++    char* pszHome = getenv("SNAP_USER_COMMON");
```

Comment:
Thanks for elaborating on that. I think we can avoid the patch be using the `-datadir=...` command line parameter. This parameter works on `faircoind`as well as `faircoin-qt`.